master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -65,6 +65,7 @@ public abstract class AbstractMetaManager implements IMetadataHandler {
   public final Set<WorkerInfo> excludedWorkers = ConcurrentHashMap.newKeySet();
   public final Set<WorkerInfo> shutdownWorkers = ConcurrentHashMap.newKeySet();
   public final Set<WorkerInfo> workerLostEvents = ConcurrentHashMap.newKeySet();
+  public final Set<WorkerInfo> workerExcludeEvents = ConcurrentHashMap.newKeySet();
```

Comment:
Have you test the compilable problem?

Reply:
@AngersZhuuuu, I have tested the compilable problem, and the test result is that there is no compilable problem.